.github/workflows/continuous-integration.yml

```diff
@@ -10,7 +10,6 @@ on:
     - cron: '0 2 * * 1' # At 02:00 on Monday
 
 env:
-  CI: true
```

Comment:
Any reason for removing this? AFAIK, this is used to trigger CI-only tests, and send the correct telemetry to NPM.

Reply:
CI is a default environment variable in GH Actions. See https://docs.github.com/es/actions/reference/environment-variables#default-environment-variables